ow_lander/src/ow_lander/trajectory_sequence.py

```diff
@@ -83,11 +83,29 @@ def _append_trajectory(self, trajectory, planning_time):
       = list(self._get_final_joint_positions_of(trajectory))
     self._planning_time_total += planning_time
 
+  def _calculate_trajectory_distance(self, trajectory):
+    distance = 0.0
+    for i in range(len(trajectory.joint_trajectory.points) - 1):
+      point1 = trajectory.joint_trajectory.points[i].positions
+      point2 = trajectory.joint_trajectory.points[i + 1].positions
+      segment_distance = sum((x - y)**2 for x, y in zip(point1, point2)) ** 0.5
+      distance += segment_distance
+    return distance
+  
   def _plan(self):
     """Calls on MoveIt to plan the next trajectory of the sequence. If no
     trajectory is provided, the plan is constructed from
     """
-    success, trajectory, planning_time, error_code = self._group.plan()
+    best_distance = float('inf')
+    for i in range(5):
```

Comment:
A named constant, e.g. `num_samples` instead of the literal 5 would be helpful for understanding this loop.